prov/gni/src/gnix_rma.c

```diff
@@ -158,17 +158,25 @@ static void __gnix_rma_copy_chained_get_data(struct gnix_fab_req *req)
 		GNIX_INFO(FI_LOG_EP_DATA, "writing %d bytes to %p\n",
 			  tail_len, addr);
 		memcpy((void *)addr,
-		       (void *) ((uint8_t *) req->int_tx_buf + GNI_READ_ALIGN),
-		       tail_len);
+			   (void *) ((uint8_t *) req->int_tx_buf + GNI_READ_ALIGN),
```

Comment:
Is there a point to adjusting the white-space here it actually looks like the spacing is indented incorrectly now.